Tasks/ContainerBuildV0/src/utils.ts

```diff
@@ -100,7 +100,10 @@ export async function getServiceDetails() {
     };
     var serviceResponse = kubectlTool.execSync(executionOption);
 
-    if (serviceResponse && serviceResponse.stdout) {
+    if (serviceResponse && serviceResponse.stderr) {
+        tl.setResult(tl.TaskResult.Failed, serviceResponse.stderr);
+    }
+    else if (serviceResponse && serviceResponse.stdout) {
         namespace = JSON.parse(serviceResponse.stdout).metadata.namespace ? JSON.parse(serviceResponse.stdout).metadata.namespace : namespace;
```

Comment:
Repeated multiple places. Assingn to variable and use in other places.

Reply:
made changes as suggested